app/views/store.blade.php

```diff
@@ -15,7 +15,7 @@
 					<tr>
 						<td>{{{$item['name']}}}</td>
 						<td>{{{$item['description']}}}</td>
-						<td>ƒ {{{$item['price']}}}</td>
```

Comment:
this deleted one is correct. we want the gilder symbol, ƒ, to be before the price